src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/RegisterEventListenersAndSubscribersPass.php

```diff
@@ -106,6 +106,8 @@ private function addTaggedServices(ContainerBuilder $container): array
                     $refs = $managerDef->getArguments()[1] ?? [];
                     $listenerRefs[$con][$id] = new Reference($id);
                     if ($subscriberTag === $tagName) {
+                        trigger_deprecation('symfony/doctrine-bridge', '6.3', 'Passing subscribers is deprecated, use listeners instead');
```

Comment:
Using Doctrine subscribers as services is deprecated, declare listeners instead.